src/index.html

```diff
@@ -4,14 +4,190 @@
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <div class="group__input-block">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="group__input-block">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              value="yes"
+              name="loveCats"
+            > Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              value="no"
+              name="loveCats"
+            > No
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              name="favoriteColor"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="timeToBed"
+              step="2"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label
+          class="cars"
+          >
+          What are your favorite brand of cars?
+            <select
+              name="favoriteCar"
+              multiple
+              size="4"
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              min="1"
+              max="10"
+              value="0"
+              name="rateOurWork"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Additional info:</legend>
+        <div class="group__input-block">
+          <label>
+          Comments:
+            <textarea
+              type="text"
+              name="comments"
+              maxlength="1000"
+              minlength="100"
+              autocomplete="off"
+            >
+            </textarea>
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+          Would you recommend us?
+            <select name="recommendUs">
```

Comment:
Use proper indentation
```suggestion
          <label>
            Would you recommend us?
            <select name="recommendUs">
```